.ci/gcb-push-downstream.yml

```diff
@@ -30,41 +30,45 @@ steps:
       waitFor: ["checkout"]
 
     # TPG
-    - name: 'gcr.io/graphite-docker-images/bash-plus'
-      entrypoint: '/workspace/.ci/scripts/bash-plus/downstream-waiter/wait_for_commit.sh'
+    - name: 'gcr.io/graphite-docker-images/go-plus'
+      entrypoint: '/workspace/.ci/scripts/go-plus/magician/exec.sh'
       id: tpg-sync
-      waitFor: ["checkout"]
+      waitFor: ["build-magician-binary", "checkout"]
```

Comment:
nit: I think actually `checkout` isn't necessary since it's already a dependency via `build-magician-binary` - but it's not going to cause problems to have it here, so not a blocker.

Reply:
Done, removed the unnecessary ones, also in the push steps.